apps/cloud/src/app.ts

```diff
@@ -19,11 +23,47 @@ app.use(morgan('dev'));
 app.use(cors());
 // Using a body parser for JSON requests.
 app.use(bodyParser.json());
+//app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser);
+//app.use(express.static('public'));
 
 app.use('/users', userRouter);
 app.use('/messages', messageRouter);
 
 import taskRoute from "./api/task"
 app.use('/tasks', taskRoute);
 
+
+// Security spike
+import securityInit from "./passportSecurity";
+securityInit(app, userStore);
+app.use('/tasksSecured', passport.authenticate('local', { session: false }), taskRoute);
+
+
+app.use(function(req, res, next) {
+    var err = new Error('Not Found');
+    next(err);
+});
+
+if (app.get('env') === 'development') {
+    app.use(function (err: any, req: any, res: any, next: any) {
```

Comment:
Express has a type for this: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts#L29-L31

Any reason for the `any`s?